otx/algorithms/detection/adapters/mmdet/nncf/builder.py

```diff
@@ -199,8 +199,11 @@ def _get_fake_data_for_forward(nncf_config):
             compression_ctrl=compression_ctrl,
         )
     )
+    # TODO: move this to OTX task when MPA is absorbed into OTX
     remove_from_configs_by_type(custom_hooks, "CancelInterfaceHook")
     remove_from_configs_by_type(custom_hooks, "TaskAdaptHook")
+    remove_from_configs_by_type(custom_hooks, "LazyEarlyStoppingHook")
```

Comment:
Could you please tell why this and similar changes are relevant?

Reply:
As name suggests, these hooks will stop training when they find a metric is not improved. We don't need theses hooks as it is essentially a role of `AccuracyAwareRunner`. With theses hooks, training would be stopped in a way that `AccuracyAwareRunner` wouldn't expect.

Reply:
I see, thank you